README.md

```diff
@@ -376,11 +376,11 @@ alerting:
 
 > Important: Do not load balance traffic between Prometheus and its Alertmanagers, but instead point Prometheus to a list of all Alertmanagers. The Alertmanager implementation expects all alerts to be sent to all Alertmanagers to ensure high availability.
 
-### Disabling high availability
+### Turn off high availability
 
-If running Alertmanager in high availability mode is not desired, setting `--cluster.listen-address=` will prevent Alertmanager from listening to incoming peer requests.
+If running Alertmanager in high availability mode is not desired, setting `--cluster.listen-address=` prevents Alertmanager from listening to incoming peer requests.
 
-## Contributing to the Front-End
+## Contribute to the front end
```

Comment:
I would rename this section `## Contributing`, include the content from [CONTRIBUTING.md](https://github.com/prometheus/alertmanager/blob/master/CONTRIBUTING.md) before the existing text and remove `CONTRIBUTING.md`.